src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -24,6 +23,15 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Http
 {
     public abstract partial class Frame : IFrameControl
     {
+        // byte consts don't have a data type annotation so we pre-cast them
```

Comment:
For some reason it doesn't like them as consts without the cast so this is a compile error
```csharp
const byte ByteCR = '\r';
```